src/lib/canister_methods/execute_method.ts

```diff
@@ -13,7 +13,7 @@ export function executeMethod(
     returnCandidType: CandidType,
     manual: boolean
 ) {
```

Comment:
Can we make T = any in encode and decode so that we don't have to explicitly pass in <any>?

Reply:
done